cdm/core/src/test/java/ucar/nc2/ncml/TestEnhance.java

```diff
@@ -76,7 +76,7 @@ public void testStandaloneEnhance() throws IOException {
 
   @Test
   public void testStandaloneEnhanceDataset() throws IOException {
-    try (NetcdfFile ncfile = NetcdfDatasets.openDataset(dataDir + "testStandaloneNoEnhance.ncml", true, null)) {
+    try (NetcdfFile ncfile = NetcdfDatasets.openDataset(dataDir + "testStandaloneEnhance.ncml", true, null)) {
```

Comment:
Maybe @haileyajohnson can weigh in, but I think maybe this was actually intended to use `testStandaloneNoEnhance.ncml`, since the one below tests `testStandaloneEnhance.ncml`. I think this tests an enhanced dataset even with the "NoEnhance" file because of the `true` in the `openDataset`

Reply:
Changed back to the original `testStandaloneNoEnhance.ncml`.